src/components/input/index.jsx

```diff
@@ -107,6 +109,28 @@ const Input = ({
 	]
 		.filter((classString) => classString)
 		.join(" ");
+
+	if (type === FIELD_TYPES.RADIO) {
```

Comment:
You can already make a radio button with `<Input type="radio" />` before this change was made. Was that not working for some reason? Is it just to get the label to appear after the input? If the label just needs to move handling it in the original `return` would be better. This extra return clutters up the component code quite a bit.

Adding the checked attribute should be fine for all non-select input types. If no value is passed in then it won't render client side.